### PR TITLE
Add `VITALS_MINUTES_BETWEEN_HEARTBEAT_CHECKS` for frequency of vitals heartbeat checks (CU-86drtj00f).

### PR DESCRIPTION
**Pull request by micahdbak**
This PR adds an environment variable to configure the frequency of vitals heartbeat checks.

Test Plan:
- [x] Test cases pass
- [x] Dev deployment is successful
- [x] Vitals messages are sent as specified by the environment variable

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86drtj00f Buttons Rejected Twilio Messages](https://app.clickup.com/t/86drtj00f)

**Comment by micahdbak** [comment]
~ Deploying to the dev server.

**Comment by micahdbak** [comment]
Merging this branch to deploy v13.7.1 later today.

File: server/server.js [hunk 1]
```diff
@@ -52,7 +52,8 @@ if (helpers.isTestEnvironment()) {
   server = https.createServer(httpsOptions, app).listen(443)
   setInterval(vitals.checkGatewayHeartbeat, 5 * 60 * 1000)
   setInterval(vitals.checkButtonBatteries, 5 * 60 * 1000)
-  setInterval(vitals.checkButtonHeartbeat, 5 * 60 * 1000)
+  const minutesBetweenHeartbeatChecks = parseInt(helpers.getEnvVar('VITALS_MINUTES_BETWEEN_HEARTBEAT_CHECKS'), 10)
+  setInterval(vitals.checkButtonHeartbeat, minutesBetweenHeartbeatChecks * 60 * 1000)
```
Comment (trisapeace) [inline server/server.js:56]: There are two different kinds of heartbeat checks here, Buttons and Gateways. I know that you've merged this already, but could you please rename the environment variable to make it obvious which one this is?

Thanks @micahdbak !

**Pull request merged**